UI/Selenium/Steps/ConsultationRoomSteps.cs

```diff
@@ -110,5 +110,75 @@ public void ThenAllParticipantsLeaveConsultationRoom()
                 Driver.FindElement(ParticipantWaitingRoomPage.JoinPrivateMeetingButton).Displayed.Should().BeTrue();
             }
         }
+
+        [Then(@"both participants click on the leave button to leave the room")]
+        public void ThenBothParticipantsLeaveConsultationRoom()
+        {
+            var participants = _hearing.Participant.Where(p => !p.Id.ToLower().Contains("judge")).ToList();
+
+            foreach (var participant in participants)
+            {
+                var participantKey = $"{participant.Id}#{participant.Party.Name}-{participant.Role.Name}";
+                Driver = GetDriver(participantKey, _scenarioContext);
+                _scenarioContext["driver"] = Driver;
+
+                var script = "document.getElementById('leaveButton-landscape').click();";
+                var scriptExecutor = Driver as IJavaScriptExecutor;
+                scriptExecutor.ExecuteScript(script);
+                ExtensionMethods.FindElementWithWait(Driver, ConsultationRoomPage.ConfirmLeaveButton, _scenarioContext).Click();
+
+                ExtensionMethods.FindElementWithWait(Driver, ParticipantWaitingRoomPage.StartPrivateMeetingButton, _scenarioContext);
+                Driver.FindElement(ParticipantWaitingRoomPage.ParticipantDetails($"{participant.Name.FirstName} {participant.Name.LastName}")).Displayed.Should().BeTrue();
+
+                ExtensionMethods.FindElementWithWait(Driver, ParticipantWaitingRoomPage.ParticipantDetails(_hearing.Case.CaseNumber), _scenarioContext, TimeSpan.FromSeconds(Config.DefaultElementWait)).Displayed.Should().BeTrue();
+                Driver.FindElement(ParticipantWaitingRoomPage.ChooseCameraAndMicButton).Displayed.Should().BeTrue();
+                Driver.FindElement(ParticipantWaitingRoomPage.JoinPrivateMeetingButton).Displayed.Should().BeTrue();
+            }
+        }
+
+        [When(@"1st participant start a private meeting and selects 2nd participant")]
+        public void When1stParticipantStartAPrivateMeetingAndSelects2ndParticipant()
+        {
+            var participants = _hearing.Participant.Where(p => !p.Id.ToLower().Contains("judge")).ToList();
```

Comment:
See my previous comment.

Reply:
Used "party" to identify non-staff participants